.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
While trying to figure out why entry group doesn't change state to ```COLLISION``` if CNAME conflicts with other .local domain on the network, I found out that avahi-daemon provides ```with-avahi-priv-access-group=``` flag that accepts name of the user group that will have ability to change host-name without sudo. By default it's ```netdev``` group.

So I'm curious if our ```foxbox``` user will be in a special group that we can feed to avahi-daemon's ```with-avahi-priv-access-group=```, @fabricedesre? That would be really nice :)